File: src/com/jarkos/coinmarket/enums/currencies/BtcCurrencyPairEnum.java
package com.jarkos.coinmarket.enums.currencies;

public enum BtcCurrencyPairEnum {
    BTCPLN("BTC/PLN"),
    BTCEUR("BTC/EUR"),
    BTCUSD("BTC/USD"),
    BTCUSDT("BTC/USDT");

    private String value;

    BtcCurrencyPairEnum(String s) {
        this.value = s;
    }

    public String getCode(BtcCurrencyPairEnum s) {
        return s.value;
    }

    public String toString() {
        return this.value;
    }
}

File: src/com/jarkos/coinmarket/enums/currencies/BtcCurrencyPairEnum.kt
package com.jarkos.coinmarket.enums.currencies

enum class BtcCurrencyPairEnum constructor(private val value: String) {
    BTCPLN("BTC/PLN"),
    BTCEUR("BTC/EUR"),
    BTCUSD("BTC/USD"),
    BTCUSDT("BTC/USDT");

    override fun toString(): String {
        return this.value
    }
}
